Doc/library/subprocess.rst

```diff
@@ -1564,33 +1564,21 @@ runtime):
 .. _disable_vfork:
 .. _disable_posix_spawn:
 
-Disabling use of ``vfork()`` or ``posix_spawn()``
+Disabling use of ``posix_spawn()``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 On Linux, :mod:`subprocess` defaults to using the ``vfork()`` system call
 internally when it is safe to do so rather than ``fork()``. This greatly
 improves performance.
 
-If you ever encounter a presumed highly unusual situation where you need to
-prevent ``vfork()`` from being used by Python, you can set the
-:const:`subprocess._USE_VFORK` attribute to a false value.
-
-::
-
-   subprocess._USE_VFORK = False  # See CPython issue gh-NNNNNN.
-
-Setting this has no impact on use of ``posix_spawn()`` which could use
-``vfork()`` internally within its libc implementation.  There is a similar
-:const:`subprocess._USE_POSIX_SPAWN` attribute if you need to prevent use of
-that.
-
 ::
 
    subprocess._USE_POSIX_SPAWN = False  # See CPython issue gh-NNNNNN.
+   subprocess._USE_VFORK = False  # See CPython issue gh-NNNNNN.
```

Comment:
I'm not very familiar with the backwards compatibilty policy, and felt like it should have a deprecation cycle as a documented public API. It does feel like this could fall as previously "soft deprecated" (per https://peps.python.org/pep-0387/#soft-deprecation) as the docs said explicitly "don't use this", but reading that policy still need a full deprecation cycle to remove fully

Reply:
If the variable is ignored, I don't see the point of documenting it. It can be removed right now.

Reply:
> I'm not very familiar with the backwards compatibilty policy

This flag is private and so doesn't have to respect https://peps.python.org/pep-0387/ policy.

Reply:
Updated to remove entirely and then make case around number of flags match (ex. these flags -> this flag)